StRoot/StETofCalibMaker/StETofCalibMaker.h

```diff
@@ -206,19 +216,28 @@ class StETofCalibMaker: public StMaker {
     std::vector<unsigned long int> mMasterStartVec;
     std::map<int , short>          mGet4StateMap;
     std::map<int , short>          mGet4ZeroStateMap;
+    std::map<int , short>          mGet4DefaultStateMap;
+
     unsigned long int              mStateMapStart;
     unsigned long int              mStateMapStop;
     unsigned long int              mDbEntryStart;
     unsigned long int              mDbEntryStop;
     int                            mGlobalCounter;
+    bool                           mCalState;
+
+    void decodeInt( std::vector<unsigned long int> intVec ,std::map<int , short>& mGet4StateMap ,std::map<int , short>& mGet4ZeroStateMap ,std::vector<unsigned long int>& startVec ,std::vector<unsigned long int>& mMasterStartVec ,std::map<unsigned long int,vector<int>>& stateVec ,std::map<unsigned long int,vector<int>>& get4IdVec, std::map<int , short>&  mGet4DefaultStateMap);
```

Comment:
As per my other comments...
```suggestion
    void decodeInt( std::vector<unsigned long int>& intVec ,std::vector<unsigned long int>& startVec ,std::map<unsigned long int,vector<int>>& stateVec ,std::map<unsigned long int,vector<int>>& get4IdVec);
```